HopsanGUI/HcomHandler.cpp

```diff
@@ -7134,6 +7158,211 @@ void HcomHandler::evaluateExpression(QString expr, VariableType desiredType)
             return;
         }
     }
+    else if(desiredType != Scalar && desiredType != DataVector && isHcomFunctionCall("expr", expr)) {
+       QStringList args = extractFunctionCallExpressionArguments(expr);
+       if(args.size() != 1) {
+           HCOMERR("Wrong number of arguments for expr function.\n"+mLocalFunctionDescriptions.find("ddt").value().second);
```

Comment:
why add `+mLocalFunctionDescriptions.find("ddt").value().second`to the error message, in particular `ddt`

Reply:
Copy-paste error. It is intended to print the help string for the function, but of course for the <i>correct</i> function. Perhaps this can leter be wrapped into some more beautiful auxiliary function.